com.unity.cinemachine/Samples~/Shared Assets/Scripts/PositionAimTargetAndReticle.cs

```diff
@@ -47,6 +47,14 @@ void SetAimTarget(CinemachineBrain brain)
                         if (AimTargetIndicator != null)
                             AimTargetIndicator.position = brain.OutputCamera.WorldToScreenPoint(transform.position);
                     }
+                    else
+                    {
+                        // Emulate unstable aim spread
```

Comment:
Can you explain the purpose of this?

Reply:
RoadieRun camera does not have 3rdPersonAim. So when RoadieRun is live, then the aim target is not updated. So aim target remains at the last position 3rdPersonAim set it. This is definitely not good.

When the player is sprinting, their aim is either unstable or blocked. In GearsOfWar's RoadieRun, shooting is disabled when RoadieRunning.

I chose to make shooting spread all over a bit when sprinting instead of disabling shooting. I felt it's less complicated for the user this way.

Reply:
It doesn't work properly after you run any significant distance.

Reply:
In any case I'm wondering if this is the right place for this logic.  The job of this class is to position the reticle where it's told, but you are abusing it to define game logic.  It's not good architecture.

Reply:
What do you mean?

Reply:
I see now

Reply:
Now it's fixed.

Reply:
It still isn't right.  The spread is not correct.  I'm gong to jump on this and fix it, as well as the Fire event thing.

Reply:
What do you suggest?

Should I not set any value, or not a randomized value?
I could revert this change, and disable shooting while sprinting instead.

Reply:
See my PR https://github.com/Unity-Technologies/com.unity.cinemachine/pull/765